x-pack/plugins/apm/server/routes/agent_explorer/fetch_agents_latest_version.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { Logger } from '@kbn/core/server';
+import { AgentName } from '../../../typings/es_schemas/ui/fields/agent';
+import { fetchWithTimeout } from '../../lib/helpers/fetch_with_timeout';
+
+const bucketUrl =
+  'https://8f83orp5pc.execute-api.eu-west-1.amazonaws.com/test/0_0_1';
```

Comment:
Can we make this configurable with a default value?

Reply:
Are you referring to `kibana.yml`? or from the UI?

Reply:
I think kibana.yml makes sense here, as it's more of an admin feature. but as I mentioned on Slack yesterday, maybe there should not be a default (yet), and we disable the feature if there is no value set for it.

Reply:
Since we have the bucket now ready, do you still think it will be valuable to extract this into a setting?

Reply:
Yes. On-prem users might need to change it, e.g. route it via a proxy to satisfy firewall policies. Also, I think it should be possible to disable it entirely, without causing network errors.

Reply:
Addressed in [35defe2](https://github.com/elastic/kibana/pull/153643/commits/35defe2c561b86817b00dbefdbb5587bad86329c).

Reply:
Disabling feature addressed in [a5f7f77](https://github.com/elastic/kibana/pull/153643/commits/a5f7f77f7e7fd47c551f42fbb655ee83d22873a2).